src/nvim/api/win_config.c

```diff
@@ -232,7 +232,20 @@ void nvim_win_set_config(Window window, Dict(float_config) *config, Error *err)
   if (!win) {
     return;
   }
+
   bool new_float = !win->w_floating;
+
+  // hide a float window
+  if(config->hide) {
```

Comment:
could be part of `parse_float_config` , so it also works in nvim_open_win

Reply:
Is it allowed to set hide in nvim_open_win :thinking:  If set, will any errors pop up?

Reply:
Yes and yes. It is the same as calling set_config right after open_win, and it has no effect on errors